examples/distributed/graphsage/node_classification.py

```diff
@@ -227,36 +292,23 @@ def run(args, device, data):
                         if th.cuda.is_available()
                         else 0
                     )
+                    sample_speed = np.mean(iter_tput[-args.log_every :])
+                    mean_step_time = np.mean(step_time[-args.log_every :])
                     print(
-                        "Part {} | Epoch {:05d} | Step {:05d} | Loss {:.4f} | "
-                        "Train Acc {:.4f} | Speed (samples/sec) {:.4f} | GPU "
-                        "{:.1f} MB | time {:.3f} s".format(
-                            g.rank(),
-                            epoch,
-                            step,
-                            loss.item(),
-                            acc.item(),
-                            np.mean(iter_tput[3:]),
-                            gpu_mem_alloc,
-                            np.mean(step_time[-args.log_every :]),
-                        )
+                        f"Part {g.rank()} | Epoch {epoch:05d} | Step {step:05d}"
+                        f" | Loss {loss.item():.4f} | Train Acc {acc.item():.4f}"
+                        f" | Speed (samples/sec) {sample_speed:.4f}"
+                        f" | GPU {gpu_mem_alloc:.1f} MB | "
+                        f"Mean step time {mean_step_time:.3f} s"
```

Comment:
Just to confirm, will **different** output formats cause an anomaly during the final test? Perhaps appropriate modifications are also needed in the test section.

Reply:
what does `different output formats` mean?

Reply:
Previously, @frozenbugs  mentioned that when modifying examples, it's important to ensure that the output messages remain unchanged, otherwise it might cause issues with testing. Here, `time` has been changed to `Mean step time`, and I'm not sure if this is something to be concerned about.

Reply:
Yes, that could be an issue. But here, our benchmark does not rely on or parse these line output log. So it's ok to change `time` to `Mean step time` here.